mslice/plotting/plot_window/plot_options.py

```diff
@@ -23,17 +41,11 @@ def __init__(self, current_config):
                 legend_widget = LegendSetter(self, legend['text'], legend['handle'], legend['visible'])
                 self.verticalLayout.addWidget(legend_widget)
                 self._legend_widgets.append(legend_widget)
-        if None not in current_config.x_range:
-            self.lneXMin.setText(str(current_config.x_range[0]))
-            self.lneXMax.setText(str(current_config.x_range[1]))
-        if None not in current_config.y_range:
-            self.lneYMin.setText(str(current_config.y_range[0]))
-            self.lneYMax.setText(str(current_config.y_range[1]))
 
     @staticmethod
     def get_new_config(current_config):
         dialog = PlotOptionsDialog(current_config)
-        dialog_accepted= dialog.exec_()
+        dialog_accepted=dialog.exec_()
```

Comment:
This is really picky but as I have other formatting comments then can we follow pep8 and out a spaces around the `=` here?